packages/core/src/utils/get-packages.ts

```diff
@@ -44,15 +46,15 @@ export const getPackages = async (
   return packages;
 };
 
-const defaultEntry = 'src/index.ts';
-const otherEntries = 'src/entries';
-
 interface GetPackageEntryPointsOpts {
   packageRoot: string;
 }
 export const getPackageEntryPoints = async ({
   packageRoot,
 }: GetPackageEntryPointsOpts): Promise<PackageEntryPoint[]> => {
+  const defaultEntry = 'src/index.ts';
+  const otherEntries = 'src/entries';
```

Comment:
These can be kept outside the function right? They're still constants.

Reply:
They sure can be. I moved them inside the function because they're only used here and to have them closer to the usage site.